docs/self-managed/platform-deployment/helm-kubernetes/platforms/redhat-openshift.md

```diff
@@ -269,7 +269,7 @@ To use these routes for the Zeebe Gateway, configure this through Ingress as wel
 
 ### Alternatives
 
-An alternative is to install the Ingress Controller of choice and use this instead; for example, [NGINX](https://www.redhat.com/en/blog/using-nginx-ingress-controller-red-hat-openshift).
+An alternative is to install the Ingress Controller of choice and use this instead. This may require additional adjustments on the Camunda 8 ingress since it heavily focuses on the [ingress-nginx Controller](https://github.com/kubernetes/ingress-nginx). Do not confuse the controller with the Red Hat endorsed [NGINX Ingress Controller](https://www.redhat.com/en/blog/using-nginx-ingress-controller-red-hat-openshift). The latter would require additional adjustments on the Camunda 8 ingress and `NGINX Ingress Controller` to make `grpc` and `http2` work, please have a look at this [example and the prerequisites](https://github.com/nginxinc/kubernetes-ingress/blob/main/examples/ingress-resources/grpc-services/README.md).
```

Comment:
I would rephrase this a bit for better clarity:

```suggestion
An alternative to using [routes](https://docs.openshift.com/container-platform/4.14/networking/routes/route-configuration.html) is to install and use one of the [Kubernetes Ingress Controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) instead. The de-facto standard in the industry is the [ingress-nginx Controller](https://github.com/kubernetes/ingress-nginx). 

Do not confuse the [ingress-nginx controller](https://github.com/kubernetes/ingress-nginx) with the [NGINX Ingress Controller](https://www.redhat.com/en/blog/using-nginx-ingress-controller-red-hat-openshift) that is endorsed by Red Hat for usage with OpenShift.

If you use [NGINX Ingress Controller](https://www.redhat.com/en/blog/using-nginx-ingress-controller-red-hat-openshift), you would require additional adjustments done to the Camunda 8 ingress objects and the NGINX Ingress Controlle itself, to make `gRPC` and `HTTP/2` connections work. In that case, please refer to the [example and the prerequisites](https://github.com/nginxinc/kubernetes-ingress/blob/main/examples/ingress-resources/grpc-services/README.md).
```

Reply:
I'd also recommend using something like a [warning admonition](https://docusaurus.io/docs/markdown-features/admonitions) for the part that brings attention to the difference between the nginx-ingress and ingress-nginx (as otherwise a reader may not pay attention to it - the naming difference between the two is minimal)

Reply:
![image](https://github.com/camunda/camunda-docs/assets/9141483/981beec1-6566-45cf-a9e5-76ef579f01a0)